contracts/CErc20.sol

```diff
@@ -212,6 +216,10 @@ contract CErc20 is CToken, CErc20Interface {
         // Underlying is the gauge token like rETH-THETA-gauge
         RBN_MINTER.mint(underlying)
 
+        uint256 toDistribute = RBN.balanceOf(address(this));
+
+        RBN.approve(rewardsDistributor, toDistribute)
```

Comment:
you don't want to do an infinite approval once, like in `becomeImplementationData`?